### ISSUE TITLE
Extensions to country FIT Count downloads

### ISSUE THREAD

**Issue by DavidRoy** [BiologicalRecordsCentre/SPRING#19]
links to #8 

Once the regional co-ordinator functionality is available, review potential extensions to downloads.
- [ ] User download.  List of users for a country and their profile settings (set on app registration) - 'I allow to be contacted'; 'Your identification experience. Include email address where 'I allowed to be contacted' is set to Yes
- [ ] consider options to migrate some reports to use ElasticSearch for improved performance

**Comment by andrewvanbreda** [comment]
Hi @kazlauskis 
Does the app already capture the information in the first point above, is that information synced to Warehouse attributes?

**Comment by kazlauskis** [comment]
Yes, the app captures this information and stores it in the user profile on the FIT Drupal site. It is not copied to warehouse though.

**Comment by andrewvanbreda** [comment]
@kazlauskis 
Ok thanks for this info. 
Actually if it is a user download only, that should be ok as will be a Drupal view. 
Will come back to this when ready to implement.
